.buildkite/llvm-apple-pipeline.yml

```diff
@@ -7,16 +7,14 @@ env:
   DOCKER_REGISTRY: "docker.elastic.co"
   STAGING_IMAGE: "${DOCKER_REGISTRY}/observability-ci"
   MAKEFILE: "go/llvm-apple"
-  CHANGESET_FILE: ".buildkite/scripts/changeset/llvm-apple"
-  DOCKER_FILTER_REF: "*/*/golang-crossbuild:llvm-apple*"
   BUILDX: "0"
 steps:
   - label: ":linux: Build LLVM Apple / Ubuntu X86_64 - {{matrix.debianVersion}}"
-    key: "build-ubuntu-x86"
+    key: "build-ubuntu-x86-llvm-apple"
```

Comment:
fix the current issue when not able to run the llvm and didn't configure the access to the Google Bucket. Although this won't be needed as part of:
- https://github.com/elastic/golang-crossbuild/pull/610